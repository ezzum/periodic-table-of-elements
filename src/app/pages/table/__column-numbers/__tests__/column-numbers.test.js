import { shallow } from 'enzyme';
import ColumnNumbers from '../column-numbers';

describe('ColumnNumbers tests', () => {
  it('snapshot test', () => {
    const wrapper = shallow(<ColumnNumbers />);
    expect(wrapper).toMatchSnapshot();
  });
});
