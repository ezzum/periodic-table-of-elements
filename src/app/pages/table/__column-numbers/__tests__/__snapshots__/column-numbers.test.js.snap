// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColumnNumbers tests snapshot test 1`] = `ShallowWrapper {}`;
